### ISSUE TITLE
Cart issue

### ISSUE THREAD

**Issue by lekiend** [wpbackoffice/woocommerce-thumbnail-input-quantities#4]
If I change quantities in the catalog and click on "Add to card", it adds only a single step in the card. 
example: If a product is only available by steps of "7", if I select "14" and click on "Add to Cart", only seven unit are added in the cart. Please check on http://c.visitonweb.be/en/product-category/products/.
Thanks
Dimitri

**Comment by jtwiest** [comment]
This is most likely a theme conflict, try deactivating your theme and see if it works.

**Comment by lekiend** [comment]
I tried adding 2x22 (44) items to the cart.
With twenty twelve, I get a RED Woocommerce alert message while adding to the cart.

You must add a minimum of 22 my_product's to your cart.

It was working before an update.
Any idea ?

**Comment by lekiend** [comment]
Within an individual product, it works correcly.

**Comment by jtwiest** [comment]
Do you have javascript enabled? What the plugin does is when your mouse leaves the input box it puts the input box value as the "data-attribute" value tied to the add to cart button. When your link was live I checked and the data-attribute values wasn't being updated, which is what caused the error. 

Also, what update did you make to break it?

Thanks.

**Comment by lekiend** [comment]
Yes, javascript is enabled. I don't know exactly what update broke your plugin. Maybe WP 3.9 or Woocommerce 2.1.9 ?
I strongly believe you have an issue in your code :-(

New URL: www.covadis.org

Dimitri

**Comment by jtwiest** [comment]
Found it, your theme is adding an extra div which is conflicting with my javascript selector. Around your add to cart button you have this: 

```
<input type="number" name="thumbnail-quantity" class="thumbnail-quantity quantity" min="22" value="22" step="22" "="">
    <div class="product-actions">
        <a data-quantity="22" href="/categorie-produit/produits-cat/?add-to-cart=600" rel="nofollow" data-product_id="600" data-product_sku="75.022" class="theme_button white button add_to_cart_button product_type_simple"><span>Ajouter au panier</span></a>
    </div>
</li>
```

If you remove the product-actions div you should be good to go.

**Comment by lekiend** [comment]
Thank you for your answer.
This DIV is used to be able to change button CSS style.
Could you add something in your code to accept DIV around the button link ?
I guess many themes does those kinds of things.
Dimitri

**Comment by lekiend** [comment]
I guess it is possible to target the "data-quantity" value with jquery with or without DIV around the link.

**Comment by jtwiest** [comment]
Yeah I'll have to revisit the code. Hopefully will be able to get to it soon. If you have any js experience and want to take a crack at it I'd love the help!
